tests/locator_test.go

```diff
@@ -91,3 +91,21 @@ func TestLocatorCheck(t *testing.T) {
 		require.Panics(t, func() { input.IsChecked(nil) }, "should not select multiple elements")
 	})
 }
+
+func TestLocatorIsEditable(t *testing.T) {
+	tb := newTestBrowser(t, withFileServer())
```

Comment:
Why is the `tparallel` linter broken again? :facepalm:

This test should have `t.Parallel()`, while the subtests probably shouldn't because they're sharing the same browser, `goja.Runtime`, etc., but then `tparallel` should complain about both cases, and we'd have to `nolint` it for the subtests... Ugh.

Reply:
No idea 🙄 You know we had many problems because of parallel tests back then (#141) and decided not to use them. Are you that we should enable it?

Reply:
I guess I'm just venting about `golangci-lint` being unreliable, as usual... If `tparallel` is not reporting, how can we be sure that other linters are? :disappointed:

It's fine if you want to not parallelize these tests; they run quick enough and we won't notice them. But I think by default we should add `t.Parallel()` anywhere where it makes sense, and deal with flakiness once we run into it. Otherwise we won't know it's there, and we might as well disable the `tparallel` linter.

Reply:
Right... I updated #366 for parallel tests. I'll make the tests parallel when it makes sense.